js/viz/chart.d.ts

```diff
@@ -1206,8 +1206,8 @@ export interface dxChartOptions extends BaseChartOptions<dxChart> {
 export interface dxChartArgumentAxis extends dxChartCommonAxisSettings {
     /**
      * @docid dxChartOptions.argumentAxis.aggregateByCategory
-     * @default false
-     * @public
```

Comment:
y do we remove `public` here?

Reply:
I guess the reason for removing it is that the prop became deprecated.

Reply:
Please check that components for all platforms still have this property.

Reply:
I checked metadata generation - it still exists in `Declarations.json`, `NGMetaData.json` and `StrongMetaData.json`